safedelete/__init__.py

```diff
@@ -16,4 +16,3 @@
 ]
 
 __version__ = "1.0.1dev"
-default_app_config = 'safedelete.apps.SafeDeleteConfig'
```

Comment:
It is still needed for Django < 3.2, but we don't do anything in the app config so it should be all right.

Reply:
Actually, `default_app_config` was only "needed" for Django <= 1.7, [since 1.8 it's deprecated](https://docs.djangoproject.com/en/1.8/ref/applications/#configuring-applications).